src/lib/Specification/Content/ContentDraftHasConflict.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Specification\Content;
+
+use eZ\Publish\API\Repository\Exceptions\UnauthorizedException;
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use EzSystems\EzPlatformAdminUi\Specification\AbstractSpecification;
+
+class ContentDraftHasConflict extends AbstractSpecification
+{
+    /** @var ContentService */
+    private $contentService;
+
+    /**
+     * @param ContentService $contentService
+     */
+    public function __construct(ContentService $contentService)
+    {
+        $this->contentService = $contentService;
+    }
+
+    /**
+     * Checks if Content has draft conflict.
+     *
+     * @param ContentInfo $contentInfo
+     *
+     * @return bool
+     *
+     * @throws UnauthorizedException
+     */
+    public function isSatisfiedBy($contentInfo): bool
+    {
+        $versions = $this->contentService->loadVersions($contentInfo);
+
+        foreach ($versions as $checkedVersionInfo) {
+            if ($checkedVersionInfo->versionNo > $contentInfo->currentVersionNo && $checkedVersionInfo->isDraft()) {
```

Comment:
We were just discussing with @mikadamczyk should this check be unified with that one in VersionsDataset::getConflictedDraftVersions(). Any opinions?